.ci/append-to-github-output.sh

```diff
@@ -5,7 +5,7 @@ set -e
 NAME=$1
 VALUE=$2
 
-if [[ -z "$NAME" || -z "$VALUE" ]]; then
```

Comment:
Instead of asserting that 2 variables with non-zero length are passed, we can assert that just 2 variables are passed.
https://github.com/stoyanK7/checkstyle/actions/runs/4658168765/jobs/8243534520#step:6:29
```
Run grep "^Diff Regression projects:" text | cat > temp
not all parameters are set
Usage:  <variable name> <variable value>
```

There are checks below to ensure that those strings are not-empty.
https://github.com/checkstyle/checkstyle/blob/1346ff94acef40b7772b5170beeb5fbc5d5b7d5a/.github/workflows/diff-report.yml#L118-L135